tests/modules/programs/vim-vint/basic-configuration.nix

```diff
@@ -0,0 +1,31 @@
+{ config, pkgs, ... }:
+
+{
+  config.programs.vim-vint = {
+    enable = true;
+    settings = {
+      cmdargs = {
+        severity = "error";
+        color = true;
+        env = { neovim = true; };
+      };
+      policies = {
+        ProhibitEqualTildeOperator.enabled = false;
+        ProhibitUsingUndeclaredVariable.enabled = false;
+        ProhibitAbbreviationOption.enabled = false;
+        ProhibitImplicitScopeVariable.enabled = false;
+        ProhibitSetNoCompatible.enabled = false;
+      };
+    };
+    config.nmt.script = let
+		configDir = if pkgs.stdenv.isDarwin then
+		  "home-files/Library/Application Support"
```

Comment:
This would only be needed if we had a similar test in the module, but we always use `xdg.configFile`.